scout/server/blueprints/variants/controllers.py

```diff
@@ -1004,11 +1004,11 @@ def generate(header, lines):
     for variant in variant_objs.limit(EXPORTED_VARIANTS_LIMIT):
         variant_line = []
         variant_line.append(str(variant.get("variant_rank", "")))  # index
-        variant_line.append(variant.get("str_repid"))  # Repeat locus
+        variant_line.append(variant.get("str_repid", ""))  # Repeat locus
```

Comment:
This was the actual fix to #5034, but the `get_str_mc` variant returning None can cause trouble I think